hack/cherry-pick.sh

```diff
@@ -80,9 +81,14 @@ if [[ $? -ne 0 ]]; then
   exit 1
 fi
 
+commit_message="UPSTREAM: $pr: Cherry-picked"
+if [ "$repo" != "k8s.io/kubernetes" ]; then
+  commit_message="UPSTREAM: $repo: $pr: Cherry-picked"
```

Comment:
@liggitt here's your chance.  Shall we update to indicate which ones were merged at the time of picking?